modules/ROOT/pages/ch2-shared-space-about.adoc

```diff
@@ -0,0 +1,29 @@
+= Shared Spaces
+
+*Last update* 16-Mar-22
+
+A _shared space_ is an elastic cloud of resources that includes Mule instances running in
+a multi-tenant environment. 
+CloudHub 2.0 provides one shared space in each supported xref:ps-gather-setup-info.adoc#private-network-region[region], to which you deploy your integration applications.
+
+Deploying to a shared space requires no setup or maintenance of the underlying infrastructure.
+
+You can deploy apps to the shared space in a region if:
+
+* You don't require isolation from the public cloud.
+* Your apps don't need to connect to an on-premises data center.
+* Your apps can use the `cloudhub.io` domain name (rather than a vanity domain name).
+* You don't need to configure custom certificates.
+// * You don't need to forward logs to an external services.
+
+Any of the following features and functionality requires a private space: 
+
+* Single-tenancy for your apps
```

Comment:
Is single-tenancy the right term? One private space can be shared by multiple sub orgs.

Reply:
@vli63127 - I understand tenancy to apply to customers. So, a private space serves one customer and is not shared between customers. Is this incorrect? @xuanshi

Reply:
No change.